frontend/device/pocketbook/device.lua

```diff
@@ -144,6 +144,15 @@ local PocketBook626 = PocketBook:new{
     emu_events_dev = "/var/dev/shm/emu_events",
 }
 
+-- PocketBook Basic Touch
+local PocketBook624 = PocketBook:new{
+    isTouchDevice = yes,
+    hasKeys = yes,
+    hasFrontlight = no,
```

Comment:
You need to add the definition up above. :-)

Reply:
kind of superfluous isn't it? Why not use the boolean values directly ;)?

Reply:
Making them functions probably offers more flexibility. It might be much more elegant this way than some super long stuff in Device:init(). Or perhaps it was simply felt that it looks clearer than `has_frontlight`. /shrugs :-)